[2025-09-20T11:59:02.391+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manual_full_load.load_dim_store manual__2025-09-20T09:58:58.018672+00:00 [queued]>
[2025-09-20T11:59:02.399+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manual_full_load.load_dim_store manual__2025-09-20T09:58:58.018672+00:00 [queued]>
[2025-09-20T11:59:02.401+0200] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-09-20T11:59:02.411+0200] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): load_dim_store> on 2025-09-20 09:58:58.018672+00:00
[2025-09-20T11:59:02.414+0200] {standard_task_runner.py:57} INFO - Started process 6847 to run task
[2025-09-20T11:59:02.417+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'manual_full_load', 'load_dim_store', 'manual__2025-09-20T09:58:58.018672+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/manual_full_load.py', '--cfg-path', '/tmp/tmp8ow0am7p']
[2025-09-20T11:59:02.418+0200] {standard_task_runner.py:85} INFO - Job 79: Subtask load_dim_store
[2025-09-20T11:59:02.458+0200] {task_command.py:410} INFO - Running <TaskInstance: manual_full_load.load_dim_store manual__2025-09-20T09:58:58.018672+00:00 [running]> on host 1102361e702e
[2025-09-20T11:59:02.531+0200] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='manual_full_load' AIRFLOW_CTX_TASK_ID='load_dim_store' AIRFLOW_CTX_EXECUTION_DATE='2025-09-20T09:58:58.018672+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-20T09:58:58.018672+00:00'
[2025-09-20T11:59:02.532+0200] {sql.py:265} INFO - Executing: 
        INSERT INTO dwh.dim_store (store_id, store_name, region)
        SELECT DISTINCT store_id, 'Unknown', 'Unknown'
        FROM staging.sales_stg
        ON CONFLICT (store_id) DO NOTHING;
        
[2025-09-20T11:59:02.545+0200] {base.py:73} INFO - Using connection ID 'postgres_retail' for task execution.
[2025-09-20T11:59:02.589+0200] {base.py:73} INFO - Using connection ID 'postgres_retail' for task execution.
[2025-09-20T11:59:02.594+0200] {sql.py:375} INFO - Running statement: 
        INSERT INTO dwh.dim_store (store_id, store_name, region)
        SELECT DISTINCT store_id, 'Unknown', 'Unknown'
        FROM staging.sales_stg
        ON CONFLICT (store_id) DO NOTHING;
        , parameters: None
[2025-09-20T11:59:02.596+0200] {sql.py:384} INFO - Rows affected: 0
[2025-09-20T11:59:02.605+0200] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=manual_full_load, task_id=load_dim_store, execution_date=20250920T095858, start_date=20250920T095902, end_date=20250920T095902
[2025-09-20T11:59:02.637+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-09-20T11:59:02.697+0200] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
